### PR TITLE
fix(BUX-304): do not relay flag should be true 

### PR DESCRIPTION
**Pull request by sirdeggen**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/pulse/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/pulse/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [x] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: transports/p2p/server.go [hunk 1]
```diff
@@ -325,17 +325,17 @@ func (sp *serverPeer) OnVersion(_ *peer.Peer, msg *wire.MsgVersion) *wire.MsgRej
...
-		missingServices := wantServices & ^msg.Services
-		sp.server.p2pConfig.Logger.Debugf("Rejecting peer %s with services %v due to not "+
-			"providing desired services %v", sp.Peer, msg.Services,
-			missingServices)
-		reason := fmt.Sprintf("required services %#x not offered",
-			uint64(missingServices))
-		return wire.NewMsgReject(msg.Command(), wire.RejectNonstandard, reason)
-	}
+	// This would allow rejection of non full node peers, but we don't want any of the services so no reason to reject outbound peers.
+	// wantServices := wire.SFNodeNetwork
```
Comment (sirdeggen) [inline transports/p2p/server.go:329]: note: we do not need to connect to full nodes to get headers. This should allow Pulse instances to connect with one another and (maybe) even send eachother header messages.

Comment (dorzepowski) [reply transports/p2p/server.go:329]: If we don't need it then I would suggest to remove this code entirely instead of commenting it out.

**Comment by sirdeggen** [comment]
@dorzepowski looks like this works, with a LOT less tx traffic in the logs. Could you please merge / release if things also work for you, and maybe update the included database?

**Comment by dorzepowski** [comment]
> @dorzepowski looks like this works, with a LOT less tx traffic in the logs. Could you please merge / release if things also work for you, and maybe update the included database?
Yes, I will take care of that today.

**Review by dorzepowski — APPROVED** [review]

**Pull request merged**